server/database/models/registrationStep.js

```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const registrationStep = new Schema({
+  title: { type: String, required: true },
+  details: { type: String, required: true },
+});
+
+module.exports = mongoose.model('regStep', registrationStep);
```

Comment:
```suggestion
module.exports = mongoose.model('registrationStep', registrationStep);
```

Reply:
i think we can export it by this name

Reply:
yes but I see this name not suitable, and using abbreviations is not correctly all the time .

Reply:
@ahmadysalah I think we talked about this and that this is going to be the name of the model